ui/app/clinical/consultation/directives/tooltip.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+angular.module("bahmni.clinical").directive("tooltip", ['$timeout', function ($timeout) {
+    return {
+        restrict: "A",
+        link: function (scope, element, attrs) {
+            var tooltip = angular.element('<div class="tooltip">');
+            tooltip.text(attrs.tooltip);
+
+            tooltip.css({
+                position: "absolute",
+                top: element.offset().top + element.height(),
+                left: element.offset().left + element.width() / 2 - 70
```

Comment:
Why is this calculation here? Can u add a comment or function to make it explicit?